tests/unit/test_zarrio.py

```diff
@@ -86,32 +89,6 @@ def setUp(self):
         self.store = [DirectoryStore(p) for p in self.store_path]
 
 
-#########################################
-#  TempStore tests
-#########################################
-class TestZarrWriterTempStore(BaseTestZarrWriter):
-    """Test writing of builder with Zarr using a custom TempStore"""
-    def setUp(self):
-        super().setUp()
-        self.store = TempStore()
-        self.store_path = self.store.path
-
-
-class TestZarrWriteUnitTempStore(BaseTestZarrWriteUnit):
-    """Unit test for individual write functions using a custom TempStore"""
-    def setUp(self):
-        self.store = TempStore()
-        self.store_path = self.store.path
-
-
-class TestExportZarrToZarrTempStore(BaseTestExportZarrToZarr):
-    """Test exporting Zarr to Zarr using TempStore."""
-    def setUp(self):
-        super().setUp()
-        self.store = [TempStore() for i in range(len(self.store_path))]
-        self.store_path = [s.path for s in self.store]
```

Comment:
I think it's Ok to remove TempStore. Maybe we declare it as deprecated and raise a DeprecationWarning in ZarrIO.__init__ , so that we can remove it with the next main release.